schemas/context/profile-privacy.schema.json

```diff
@@ -1,89 +1,46 @@
-              "type": "array",
-              "items": {
-                "type": "object",
-                "properties": {
-                  "xdm:optOutType": {
-                    "title": "Opt-out type",
-                    "type": "string",
-                    "description": "The type of opt-out.",
-                    "enum": [
-                      "general_opt_out",
-                      "sales_sharing_opt_out"
-                    ],
-                    "meta:enum": {
-                      "general_opt_out": "General opt-out",
-                      "sales_sharing_opt_out": "Sales sharing opt-out"
-                    }
-                  },
-                  "xdm:optOutValue": {
-                    "title": "Opt-out value",
-                    "type": "string",
-                    "description": "The value of opt-out signal.",
-                    "enum": [
-                      "not_provided",
-                      "pending",
-                      "in",
-                      "out"
-                    ],
-                    "meta:enum": {
-                      "not_provided": "Not provided",
-                      "pending": "Pending verification",
-                      "in": "Opt-in",
-                      "out": "Opt-out"
-                    }
-                  },
-                  "xdm:timestamp": {
-                    "title": "Opt-out timestamp",
-                    "description": "Timestamp of the received opt-out signal.",
-                    "type": "string",
-                    "format": "date-time"
-                  }
-                }
-              }
-            }
-          }
+  "definitions":{ 
+    "profile-privacy":{ 
+      "properties":{ 
+        "xdm:optOutConsentLevel":{ 
+          "type":"object",
+          "title":"Global Privacy/Marketing Preference Values",
+          "description":"Global (User/Profile-level) Privacy/Personalization/Marketing Preferences.",
+          "ref": "https://ns.adobe.com/xdm/context/consentpreferences"
         },
-        "xdm:identityPrivacyInfo": {
+        "xdm:identityPrivacyInfo":{
           "title": "Identity level privacy information",
           "description": "Captured privacy information for identities in profile.",
           "type": "object",
           "meta:xdmType": "map",
-          "additionalProperties": {
+          "additionalProperties" : {
             "title": "User namespace",
             "type": "object",
             "meta:xdmType": "map",
             "additionalProperties": {
               "type": "object",
               "title": "User Identity",
               "properties": {
+                "xdm:consentsAndPreferences":{
+                  "title": "Device/Identity-specific Consents, Personalization and Marketing Preferences",
+                  "ref": "https://ns.adobe.com/xdm/context/consentpreferences"
+                },
                 "xdm:identityIABConsent": {
```

Comment:
here xdm:identityIABConsent and xdm:consentsAndPreferences are parallel but in associated example xdm:identityIABConsent seems to be child of xdm:consentsAndPreferences.